web/src/composebox_typeahead.ts

```diff
@@ -1411,6 +1411,14 @@ export function initialize_compose_typeahead($element: JQuery<HTMLTextAreaElemen
     );
 }
 
+export let composebox_topic_typeahead: Typeahead<string>;
+
+export function maybe_show_topic_box_typeahead(): void {
+    if (!realm.realm_mandatory_topics && $("input#stream_message_recipient_topic").val() === "") {
+        composebox_topic_typeahead.lookup(false);
+    }
+}
```

Comment:
I'm not sure where in the code the bug is, but the `C` keyboard shortcut isn't working properly in the development environment for me; it seems to start in general chat, rather than starting a new topic.

![image](https://github.com/user-attachments/assets/bec4b871-94e0-467f-9927-42ef32cdac0b)

Reply:
```diff
-        if (opts.trigger === "clear topic button") {
+        if (opts.trigger === "clear topic button" || opts.trigger === "compose_hotkey") {
            return "input#stream_message_recipient_topic";
        }
```

Had to add this to fix.

Reply:
Yeah, that sounds right.